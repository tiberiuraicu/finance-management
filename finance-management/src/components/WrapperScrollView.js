import React from "react";
import { StyleSheet, ScrollView, View } from "react-native";
import { BACKGROUND_COLOR } from "../util/constants/styleConstants";

const WrapperScrollView = (props) => {
  return (
    <View style={styles.wrapperView}>
      <ScrollView
        contentContainerStyle={{ flexGrow: 1 }}
        style={styles.wrapperScrollView}
      >
        {props.children}
      </ScrollView>
    </View>
  );
};

const styles = StyleSheet.create({
  wrapperScrollView: {},
  wrapperView: {
    height: "100%",
    backgroundColor:BACKGROUND_COLOR
  },
});

export default WrapperScrollView;
