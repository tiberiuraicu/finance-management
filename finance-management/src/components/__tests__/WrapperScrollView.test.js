import WrapperScrollView from "../WrapperScrollView";
import { shallow } from 'enzyme';

describe('WrapperScrollView component tests', () => {
    it('renders the component', () => {
      const wrapper = shallow(<WrapperScrollView />);
      expect(wrapper).toMatchSnapshot();
    });
  });