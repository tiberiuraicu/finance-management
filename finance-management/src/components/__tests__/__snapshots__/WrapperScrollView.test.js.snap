// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WrapperScrollView component tests renders the component 1`] = `ShallowWrapper {}`;
